src/main/resources/assets/createcasing/lang/ru_ru.json

```
@@ -1,232 +1,489 @@
+  "item.createcasing.vertical_refined_radiance_gearbox": "Сияющая вертикальная коробка передач",
+  "item.createcasing.vertical_shadow_steel_gearbox": "Теневая вертикальная коробка передач",
   "item.createcasing.chorium_ingot": "Хориевый слиток",
   "item.createcasing.processing_chorium": "Обработка хориевого слитка",
 
   "createcasing.ponder.tag.encased_blocks": "Create: Encased",
-  "createcasing.ponder.tag.encased_blocks.description": "Компоненты, добавленные Create Encased",
+  "createcasing.ponder.tag.encased_blocks.description": "Компоненты, добавленные Create: Encased",
 
-  "createcasing.ponder.gearbox.header": "Передачей силы вращения с помощью Коробок передач",
+  "createcasing.ponder.gearbox.header": "Передачей силы вращения с помощью коробок передач",
   "createcasing.ponder.gearbox.text_1": "Переходы между осями вращения могут быстро стать очень громоздкими",
-  "createcasing.ponder.gearbox.text_2": "Коробка Передач — это более компактный эквивалент этой установки",
+  "createcasing.ponder.gearbox.text_2": "Коробка передач — это более компактный эквивалент этой установки",
   "createcasing.ponder.gearbox.text_3": "Валы по углам поворачиваются в зеркальных направлениях",
   "createcasing.ponder.gearbox.text_4": "Прямые соединения будут реверсированны",
-  "createcasing.ponder.mechanical_mixer.header": "Обработкой предметов Механическими смешивателями",
-  "createcasing.ponder.mechanical_mixer.text_1": "С помощью Смешивателя и Чаши можно автоматизировать некоторые рецепты создания",
+  "createcasing.ponder.mechanical_mixer.header": "Обработкой предметов механическими смешивателями",
+  "createcasing.ponder.mechanical_mixer.text_1": "С помощью смешивателя и чаши можно автоматизировать некоторые рецепты создания",
   "createcasing.ponder.mechanical_mixer.text_2": "Доступные рецепты включают бесформенное создание и автоматическую варку зелий",
-  "createcasing.ponder.mechanical_mixer.text_3": "Некоторым рецептам может потребоваться тепло от Горелки всполоха",
+  "createcasing.ponder.mechanical_mixer.text_3": "Некоторым рецептам может потребоваться тепло от горелки всполоха",
   "createcasing.ponder.mechanical_mixer.text_4": "Можно использовать слот для фильтра рецепта в случае конфликта двух рецептов",
-  "createcasing.ponder.mechanical_press.header": "Обработкой предметов Механическими прессами",
+  "createcasing.ponder.mechanical_press.header": "Обработкой предметов механическими прессами",
   "createcasing.ponder.mechanical_press.text_1": "Механический пресс может обрабатывать предметы под собой",
-  "createcasing.ponder.mechanical_press.text_2": "Подаваемые предметы должны быть брошены или помещены на  под Пресс",
+  "createcasing.ponder.mechanical_press.text_2": "Подаваемые предметы должны быть брошены или помещены под пресс",
   "createcasing.ponder.mechanical_press.text_3": "Когда предметы подаются на конвейере...",
   "createcasing.ponder.mechanical_press.text_4": "...Пресс будет задерживать и прессовать их автоматически",
```

Not from your changes, but I think it should be lowercased for consistency:
```suggestion
  "createcasing.ponder.mechanical_press.text_4": "...пресс будет задерживать и прессовать их автоматически",
```